confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -114,12 +153,50 @@ class Confidence internal constructor(
     ) {
         eventSenderEngine.emit(eventName, message, getContext())
     }
+
+    private val networkExceptionHandler by lazy {
+        CoroutineExceptionHandler { _, _ ->
+            // network failed, provider is ready but with default/cache values
+        }
+    }
+
+    private fun fetch(): Job = coroutineScope.launch(networkExceptionHandler) {
+            try {
+                val resolveResponse = resolve(listOf())
+                if (resolveResponse is Result.Success) {
+                    // we store the flag anyways except when the response was not modified
+                    if (resolveResponse.data != FlagResolution.EMPTY) {
+                        diskStorage.store(resolveResponse.data)
+                    }
+                }
+            } catch (e: ParseError) {
+                throw ParseError(e.message)
+            }
+        }
+
+    fun activate() {
```

Comment:
Is there a time when we do not want to activate?

Reply:
I think no but confidence object is dumb. I prefer these actions to be explicit and set by the user and be mentioned in the user guide.